stream_alert_cli.py

```diff
@@ -123,6 +123,11 @@ def build_parser():
         help='Names of rules to test, separated by spaces'
     )
 
+    lambda_parser.add_argument(
+        '-l', '--live',
```

Comment:
I don't know how to best do this but still support the `--rules` feature, etc. Having multiple commands seems like overcomplicating this to me, especially since one command is just a boolean switch. That same logic is accomplished through the presence of a cluster value.